queries.sql

```diff
@@ -0,0 +1,121 @@
+    GROUP BY f.nome; 
+
+-- 2. Filial que vendeu mais (ordenando por qtd de venda)
+    SELECT f.nome AS NOME_FILIAL, count(v.id) AS NUM_VENDAS_MAX
+    FROM Filial f 
+        INNER JOIN Funcionario fu ON fu.filial = f.id
+        INNER JOIN Venda v ON v.id_func = fu.id
+    GROUP BY f.nome
+    ORDER BY NUM_VENDAS_MAX DESC LIMIT 1; 
+
+-- 3. Funcionario que vendeu mais
+    SELECT func.id AS ID_FUNC, count(v.id) AS NUM_VENDAS 
+    FROM Funcionario func, Venda v
+    WHERE v.id_func = func.id
+    GROUP BY v.id_func
+    ORDER BY count(v.id) DESC LIMIT 1 ; -- pensar aqui num jeito pq pode ter funcionarios com o mesmo numero de vendas
+
+-- 4. Funcionario que vendeu menos
+    SELECT func.id AS ID_FUNC, count(v.id) AS NUM_VENDAS 
+    FROM Funcionario func, Venda v
+    WHERE v.id_func = func.id
+    GROUP BY v.id_func
+    ORDER BY count(v.id) ASC LIMIT 1 ; 
+
+-- 5. Média de vendas por crédito (em cima da view)
+    SELECT avg(valor_total)
+    FROM Vendas_Cartao vc
+    WHERE vc.id_pgto = 2;
+
+-- 6. Média de vendas por débito (em cima da view)
+    SELECT avg(valor_total)
+    FROM Vendas_Cartao vc
+    WHERE vc.id_pgto = 3;
+
+-- 7. Média de vendas por dinheiro (em cima da view)
+    SELECT avg(valor_total)
+    FROM Vendas_Dinheiro;
+
+-- 8. Qual cliente comprou mais
+    SELECT c.nome AS NOME_CLIENTE, count(v.id) AS NUM_COMPRAS
+    FROM Venda v 
+        INNER JOIN Carrinho ca ON ca.id = v.id_carrinho
+        INNER JOIN Cliente c ON c.id = ca.id_cliente
+    GROUP BY c.nome
+    ORDER BY NUM_COMPRAS DESC LIMIT 1; 
+
+-- 9. Qual cliente comprou menos
+    SELECT c.nome AS NOME_CLIENTE, count(v.id) AS NUM_COMPRAS
+    FROM Venda v 
+        INNER JOIN Carrinho ca ON ca.id = v.id_carrinho
+        INNER JOIN Cliente c ON c.id = ca.id_cliente
+    GROUP BY c.nome
+    ORDER BY NUM_COMPRAS ASC LIMIT 1; 
+
+-- 10. Produtos mais pedidos (sem receita) (refrigerante, cerveja)
+    SELECT p.descricao AS NOME_PRODUTO, count(i.id) AS NUM_PEDIDOS
+    FROM Produto p
+        INNER JOIN Item i ON i.id_produto = p.id
+    WHERE p.tem_materia_prima = false
+    GROUP BY p.descricao
+    ORDER BY NUM_PEDIDOS DESC LIMIT 5; -- pega os top 5
+    
+
+-- 11. Produtos mais pedidos (com receita) (por exemplo pastel de carne)
+    SELECT p.descricao AS NOME_PRODUTO, count(i.id) AS NUM_PEDIDOS
+    FROM Produto p
+        INNER JOIN Item i ON i.id_produto = p.id
+    WHERE p.tem_materia_prima = true
+    GROUP BY p.descricao
+    ORDER BY NUM_PEDIDOS DESC LIMIT 5; -- pega os top 5
+
+-- 12. Matéria prima que gasta mais por semana
+
+-- 13. Numero de entregas por funcionario entregador
+    SELECT func.nome AS NOME_FUNCIONARIO, count(e.id) AS NUM_ENTREGAS
+    FROM Funcionario func
+        INNER JOIN Entrega e ON e.id_func = func.id
+    WHERE func.tipo = 2
+    GROUP BY func.nome;
+ 
+-- 14. Produto mais vendido no dia
+    SELECT p.descricao AS PRODUTO_MAIS_VENDIDO_HJ, count(i.id) AS NUM_PEDIDOS
+    FROM Produto p, Venda v
```

Comment:
```suggestion
    FROM Produto p
```
(já faz o inner com venda embaixo, se deixar esse aí dá erro)